ledger/participant-integration-api/src/main/scala/platform/store/backend/StorageBackend.scala

```diff
@@ -324,8 +324,8 @@ object EventStorageBackend {
       commandId: Option[String],
       workflowId: Option[String],
       eventId: EventId,
-      contractId: platform.store.dao.events.ContractId,
-      templateId: Option[platform.store.dao.events.Identifier],
+      contractId: ContractId,
+      templateId: Option[Ref.Identifier],
```

Comment:
There is also an alias for the Identifier in the package object, why not use it consistently here?

Reply:
There are more examples of that in this PR

Reply:
Our codebase is very inconsistent in the use of full aliases (e.g. `Identifier`) vs qualified types (e.g., `Ref.Identifier`). I have added a PR that changes many locations to use the aliases. It's far from complete.

To be honest, I'm not entirely sure these aliases are even a good idea. Most of them only save 4 characters (`Ref.X` vs `X`), and we have name clashes (`ParticipantId`: `com.daml.ledger.api.domain.ParticipantId` vs `com.daml.lf.data.Ref.ParticipantId`). What do you think @nmarton-da @mziolekda? Should we convert the remaining places to use the aliases from the package object, or delete the package object?

Reply:
I am ok with those aliases **because** they are platform private. I would have a lot of qualms with them if they were public.